components/blocks/EventsTimezones.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Couple questions
1. Is this hard-coded or can we edit via Tina
2. What if we dont have an event in a major city?
3. Should this be in the components/blocks/ directory?

Reply:
This is hard coded – editing it in Tina could be possible, but it's just for reference really not to be relied upon.

As per my chat with Brady I'm planning to add a warning that lets users know that they'll need to account for any daylight savings time changes when selecting a timezone as a temporary fix – and I'll make another PBI for the eventual "correct" solution. 🥱

Reply:
Quick draft PBI – https://github.com/tinacms/tina.io/issues/2422